src/main/java/org/betonquest/betonquest/Backpack.java

```diff
@@ -181,66 +181,86 @@ public Page(final int page) {
 
             // if there are other pages, place the buttons
             if (page > 1) {
-                ItemStack previous;
-                try {
-                    previous = new QuestItem(new ItemID(Config.getDefaultPackage(), "previous_button")).generate(1);
-                } catch (final ObjectNotFoundException e) {
-                    LOG.warn("Could not find item: " + e.getMessage(), e);
+                ItemStack previous = null;
+                final String previousButton = Config.getString("config.items.backpack.previous_button");
+                if (previousButton != null && !previousButton.isEmpty()) {
+                    try {
+                        previous = new QuestItem(new ItemID(null, previousButton)).generate(1);
+                    } catch (final ObjectNotFoundException e) {
+                        LOG.warn("Could not find item: " + e.getMessage(), e);
+                    } catch (final InstructionParseException e) {
+                        LOG.warn("Could not load previous button: " + e.getMessage(), e);
+                        player.closeInventory();
+                        return;
+                    }
```

Comment:
I don't think it makes sense any longer to handle the item not being found in a different way then the instruction being invalid.

It made sense back in time to do it that way as you didn't want the backpack to break if the default package was emptied. But as this clearly is no longer the case, it should instead not open the backpack so that quest authors see the issue and fix it.
```suggestion
                    try {
                        previous = new QuestItem(new ItemID(null, previousButton)).generate(1);
                    } catch (final ObjectNotFoundException e) {
                        LOG.warn("Could not find item: " + e.getMessage(), e);
                        player.closeInventory();
                        return;
                    } catch (final InstructionParseException e) {
                        LOG.warn("Could not load previous button: " + e.getMessage(), e);
                        player.closeInventory();
                        return;
                    }
```